otherlibs/Makefile.otherlibs.common

```diff
@@ -82,7 +82,7 @@ else
 endif
 
 $(LIBNAME).cmxs: $(LIBNAME).cmxa $(STUBSLIB)
-	$(CAMLOPT_CMD) -shared -o $(LIBNAME).cmxs -I . $(LIBNAME).cmxa
+	$(CAMLOPT_CMD) -shared -linkall -o $(LIBNAME).cmxs -I . $(LIBNAME).cmxa
```

Comment:
What is this supposed to achieve?

Reply:
This definitely shouldn't be there, it was the first attempt to enable `unix_exit` in `unix.cmxs`, which didn't work, so at the end after some discussion, we decided to pursue the other approach, i.e., to add it as an official function to the Unix module. We did this in parallel, see #9914. I think we can close this PR.